python/hail/expr/expressions/typed_expressions.py

```diff
@@ -2048,6 +2048,75 @@ def split(self, delim, n=None):
         else:
             return self._method("split", tarray(tstr), delim, n)
 
+    def lower(self):
+        """Returns a string of only lower-case characters.
```

Comment:
can this be something like "Returns a string with all letters converted to lowercase." or something? (Basically making it more clear that it's performing an action on the given string---I got a little confused because `s` is defined quite a ways up.) Ditto for upper.

Reply:
how about this (copied from Python stdlib docs):
Return a copy of s, but with upper case letters converted to lower case.

Reply:
sounds good! I don't have strong feelings about exact verbiage.